MonoGame.Framework/Audio/SoundEffect.cs

```diff
@@ -262,13 +262,7 @@ public static int GetSampleSizeInBytes(TimeSpan duration, int sampleRate, AudioC
         /// </remarks>
         public bool Play()
         {
-            var inst = GetPooledInstance(false);
-            if (inst == null)
-                return false;
-
-            inst.Play();
-
-            return true;
+            return Play(1f, 0f, 0f);

```

Comment:
Sorry, you are right. I though I was having problems with this, but I just tried again and it works as expected. I must've got it confused with the other problems I'm having with `CalculatePanMatrix` (issue #5054).

Do you want to keep it anyway (given it tidies up the code a bit)? Otherwise, I'll just close this.